### DISCUSSION TITLE
rules of engagement

### DISCUSSION THREAD

**Discussion by riddbtw** [aiming-taxonomy/glossary#1] [Ideas]
I spoke to Sini earlier on discord, and he brought up a lot of good points about how this should be managed.  Long story short, we're going to need to take a more rigorous approach to this vs. just submitting/merging readme changes.  here's what we have so far, apologies for the wall of text in advance...note - I manage software projects for a living so I can't help but look at things from that lens 🙃

**proposed development approach**

- migrate ghub to a ‘neutral’ (non-named) community-driven ghub, to avoid directly associating with any individual or group
- build this taxonomy as a community-driven effort with collective development or buy-in from all major individuals/groups in the aim community
- hold discussions in git surrounding proposed concepts to define and allow discussion on the concepts
- identify users who can act as mergers, should be trusted/established aim community members that meet entry criteria listed below
- use decision criteria defined below to establish consensus on concepts with divergent perspectives.  use git voting system (reacts) to establish consensus
- potentially create a discussion topic per definition? and submit PR when decision is made.  note: would like to avoid “gated” criteria for decisions so we're not collectively waiting around for someone to ‘vote’ before submitting changes to master. 

**merger entry criteria**

- vested interest in product development, outcome is to have a community-driven common understanding of aim concepts.  not someone who would be advancing their own personal interests.  note: i don't have a problem with people advancing their own interests generically (we all gotta eat), just don't think this is the place for it
- decent understanding of git concepts, technical ability to handle merge conflicts, respond to pr's, etc
- dedicated to continued development of the repo, so pr's and comments don't languish

**product goal**
- aim-community driven taxonomy - common understanding of aim concepts

**potential  stakeholders**
- voltaic
- respectfulaim
- aimlabs
- kovaaks
- pureG
- vF
- ??

**decision criteria**
suggest using something like the picture below to make decisions efficiently
![decider-and-resolution-in-slack1](https://user-images.githubusercontent.com/51275582/116946521-3191a100-ac2f-11eb-8d63-27a850eb8dc1.jpg)

**Comment by riddbtw** [comment]
totally open to other thoughts and ideas about how to approach this, I wanted to at least get a conversation started.  Other notes: Sini said that Kovaak may have already started something that could be used as a starting point

**Comment by animafps** [comment]
Amazing,
To start implementation of these proposed development approaches will start with 

- Moving the repo to a new github organization so its not associated directly to my brand
- Set out a pinned discussion for the workflow for new ideas( voting system, criteria, etc)
Discussions for discussions and submissions for new things
issues for issues with formatting, spelling/ grammar
Pull requests will be locked for maintainers 
- make a form or applications for new mergers/ maintainers 
- add some other important stuff for maintainers

As well just going to start and make drafts of the templates and framework as will be a while to get more people involved in this project

Truly thanks for helping and creating this post if you want to help out further i can add you to the github org to help with managing as clearly you have got the skills.

**Reply by riddbtw** [reply]
Great initiative! I don't mind helping support the org, I'm not a git master by any stretch but it's mostly readmes so it's fairly easy to maintain.  The only caveat is I don't consider myself the type of authority Sini was referring to, I think there's a lot more qualified people than me who have been around longer who should take those roles.  Just because it's the ideas themselves which will need to be reviewed for merge.  But if few others step up I'll be glad to do my best there.  If the discussions are fruitful the merges should be less painful.

**Reply by animafps** [reply]
> 
> 
> The only caveat is I don't consider myself the type of authority Sini was referring to, I think there's a lot more qualified people than me who have been around longer who should take those roles. 

Well yeah, thinking of more a management role to help start this up and manage adding mergers and making sure that i dont have full power rn and as your a project manager more seeing if you can help where you have the expertise.

**Reply by animafps** [reply]
> 
> 
> I think there's a lot more qualified people than me who have been around longer who should take those roles.

Who would these people be so i can try get them on board

**Reply by riddbtw** [reply]
Off the top of my head, assuming they are interested: Sini, Fallen, Hollow, Sens (from PureG), Cyn (from rA).  Maybe people like Daan, Pauer, Gored? I don't have relationships with most of the people unfortunately.  Cyn, Fallen and Sens I think I could get their buy-in fairly easily.  I'd need help with the big names like Aimlabs reps and Kovaak, any vF reps if we want to pull them in.

**Reply by riddbtw** [reply]
> > The only caveat is I don't consider myself the type of authority Sini was referring to, I think there's a lot more qualified people than me who have been around longer who should take those roles.
> 
> Well yeah, thinking of more a management role to help start this up and manage adding mergers and making sure that i dont have full power rn and as your a project manager more seeing if you can help where you have the expertise.

Sure, I'm on board.
